src/core/js/app.js

```diff
@@ -103,7 +110,7 @@ require([
         console.error(e);
     }
 
-    Adapt.mapAdaptIdsToObjects = function() {
+    function mapAdaptIdsToObjects () {
```

Comment:
I think @tomgreenfield was reverting [this change](https://github.com/tomgreenfield/adapt_framework/commit/cfa1f7acb171049a25bc46db9c01ed70224c43ab#diff-3eb546802b7afc77891e82198f76a794) since it's no longer necessary